starboard/build/config/base_configuration.gni

```diff
@@ -166,7 +166,7 @@ declare_args() {
   use_skia_next = false
 
   # Enable when using clang 16.
-  is_clang_16 = false
+  is_clang_16 = true
```

Comment:
I think we should default this to `is_clang` instead of `true`

Reply:
Yep, that's better - updated. Also updated Android and see if it builds, internal try-builds also running.